src/components/Label/Label.tsx

```diff
@@ -0,0 +1,25 @@
+import { LabelProps, Label as PatternflyLabel } from '@patternfly/react-core';
+
+import { EmptyStateSymbol } from 'components/EmptyStateSymbol/EmptyStateSymbol';
+
+interface ILabelMapperObject {
+  text: string;
+  color: LabelProps['color'];
+}
+
+export interface ILabelMapper {
+  [key: string]: ILabelMapperObject;
+}
+
+interface ILabelProps {
+  labelObject: ILabelMapperObject | undefined;
+}
+
+/**
+ * Represents a Label generated by labelMapper object.
+ *
+ * @param labelObject - the mapper object that will be used to generate the label
+ */
+export const Label = ({ labelObject }: ILabelProps) => {
```

Comment:
@patrikk0123 proposed some other name could be considered, as also PatternFly `Label` is often used directly. As it's valid point, what would you say about:

```tsx
<LabelMapper mapper={xx} />
```

Then the final structure would be:
```
components/LabelMapper/LabelMapper.tsx
components/LabelMapper/ArtifactQualityLabelMapper.tsx // apply LabelMapper suffix everywhere
...
```

cc: @DnsZhou

Reply:
done